BDArmory/BDArmory.csproj

```diff
@@ -930,4 +930,46 @@
     <PreBuildEvent>
     </PreBuildEvent>
   </PropertyGroup>
+  <PropertyGroup>
```

Comment:
This already exists in the BDArmory.csproj file with the condition
```<PostBuildEvent Condition="'$(OS)' == 'Windows_NT' ">```
which should handle all Windows builds.